{{ cookiecutter.slug }}/.github/workflows/auto-release.yaml

```diff
@@ -0,0 +1,20 @@
+{%- raw -%}
```

Comment:
Probably not necessary, since we don't use any actions templates which also use `{{ ... }}` syntax.